evoHike.Backend/Controllers/TrailsController.cs

```diff
@@ -0,0 +1,21 @@
+﻿using evoHike.Backend.Data;
```

Comment:
`using evoHike.Backend.Data`; miért van szükség? Nem láttom, hogy bárhol is használva lenne